### ISSUE TITLE
BaseLayers not available

### ISSUE THREAD

**Issue by dominikjasek** [kolebjak/react-mapycz#523]
Hi, thanks for this amazing package.

I want to allow user to set [map layer](https://api.mapy.cz/view?page=layers-auto) (ortophoto, tourist etc.) [Here](https://github.com/softopus-io/react-mapycz/blob/9e7b5672c773b4133a40ab96932a240cbb4556a6/home/MapElement.tsx#L32) you are talking about `baseLayers` but apparantely this prop doesn't exist.

Could you check it please?

Thanks

**Comment by kolebjak** [comment]
Hey, thanks I'll take a look 👍

**Comment by dominikjasek** [comment]
Hi, any updates please? 👀 🙏

**Comment by dominikjasek** [comment]
FYI, I created patch [here](https://github.com/dominikjasek/city-hunter/blob/master/patches/react-mapycz%401.2.0.patch). It is hardcoded so not very generic, but posting it here for anyone else.

**Comment by kolebjak** [comment]
Hey @dominikjasek, sorry to keep you waiting. I just merged a PR with LayoutControl, which should be what you need. You can use it as follows: 

```
<Map>
  <LayoutControl layers={[BaseLayers.HISTORIC, BaseLayers.SMART_BASE, BaseLayers.GEOGRAPHY]} top={10} left={10} />
</Map>
```

It's available in version `1.2.1`. If there's anything else you need let me know.
